bigtable/tests/integration_tests_utils.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file defines common routines and variables to run the integration tests.
+
+readonly CBT_CMD="${CBT:-${GOPATH}/bin/cbt}"
+readonly CBT_EMULATOR_CMD="${CBT_EMULATOR:-${GOPATH}/bin/emulator}"
+
+# Remove all the tables from a Cloud Bigtable instance.
+function delete_all_tables() {
+  local project_id=$1
+  shift
+  local instance_id=$1
+  shift
+
+  # If the instance is not set, just return immediately.
+  if [ -z "${instance_id}" ]; then
+    return 0
+  fi
+  ${CBT_CMD} -project "${project_id}" -instance "${instance_id}" ls |
```

Comment:
The instance may be used by other tests that may need long lived tables.  For now, can you please add a TODO for a more targeted strategy in the future?

Reply:
Note that I created a separate instance to run the integration tests so the other tests are not affected.  I also created a lock in Jenkins to represent this new instance.

In any case, I added #356 to make the integration tests smarter, and put the TODO linking to the new issue in the script.